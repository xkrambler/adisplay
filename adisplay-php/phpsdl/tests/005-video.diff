009+ SDL_VideoInit():int(-1)
010+ SDL_GetCurrentVideoDriver():bool(false)
009- SDL_VideoInit():int(0)
010- SDL_GetCurrentVideoDriver():string(%d) "%s"
012+ SDL_GetDisplayName(0):bool(false)
013+ SDL_GetDisplayBounds(0):int(-1)
014+ NULL
015+ SDL_GetNumDisplayModes(0):int(-1)
016+ SDL_GetDisplayMode(0,0):bool(false)
017+ SDL_GetDesktopDisplayMode(0):bool(false)
018+ SDL_GetCurrentDisplayMode(0):bool(false)
019+ string(0) ""
020+ NULL
021+ Segmentation fault (core dumped)
022+ 
023+ Termsig=11
012- SDL_GetDisplayName(0):string(9) "Color LCD"
013- SDL_GetDisplayBounds(0):int(0)
014- object(SDL_Rect)#1 (4) {
015-   ["x"]=>
016-   int(0)
017-   ["y"]=>
018-   int(0)
019-   ["w"]=>
020-   int(1680)
021-   ["h"]=>
022-   int(1050)
023- }
024- SDL_GetNumDisplayModes(0):int(%d)
025- SDL_GetDisplayMode(0,0):object(SDL_DisplayMode)#2 (4) {
026-   ["format"]=>
027-   int(%d)
028-   ["w"]=>
029-   int(%d)
030-   ["h"]=>
031-   int(%d)
032-   ["refresh_rate"]=>
033-   int(%d)
034- }
035- SDL_GetDesktopDisplayMode(0):object(SDL_DisplayMode)#2 (4) {
036-   ["format"]=>
037-   int(%d)
038-   ["w"]=>
039-   int(%d)
040-   ["h"]=>
041-   int(%d)
042-   ["refresh_rate"]=>
043-   int(%d)
044- }
045- SDL_GetCurrentDisplayMode(0):object(SDL_DisplayMode)#2 (4) {
046-   ["format"]=>
047-   int(%d)
048-   ["w"]=>
049-   int(%d)
050-   ["h"]=>
051-   int(%d)
052-   ["refresh_rate"]=>
053-   int(%d)
054- }
055- string(%d) "SDL_DisplayMode(%s,%d,%d,%d)"
056- NULL
057- int(0)
058- NULL
059- int(-1)
060- NULL
061- Done