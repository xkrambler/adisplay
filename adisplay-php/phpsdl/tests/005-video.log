
---- EXPECTED OUTPUT
== VideoDriver ==
int(%d)
string(%d) "%s"
bool(false)
== Video OFF ==
bool(false)
int(0)
== Video ON ==
SDL_VideoInit():int(0)
SDL_GetCurrentVideoDriver():string(%d) "%s"
SDL_GetNumVideoDisplays():int(%d)
SDL_GetDisplayName(0):string(9) "Color LCD"
SDL_GetDisplayBounds(0):int(0)
object(SDL_Rect)#1 (4) {
  ["x"]=>
  int(0)
  ["y"]=>
  int(0)
  ["w"]=>
  int(1680)
  ["h"]=>
  int(1050)
}
SDL_GetNumDisplayModes(0):int(%d)
SDL_GetDisplayMode(0,0):object(SDL_DisplayMode)#2 (4) {
  ["format"]=>
  int(%d)
  ["w"]=>
  int(%d)
  ["h"]=>
  int(%d)
  ["refresh_rate"]=>
  int(%d)
}
SDL_GetDesktopDisplayMode(0):object(SDL_DisplayMode)#2 (4) {
  ["format"]=>
  int(%d)
  ["w"]=>
  int(%d)
  ["h"]=>
  int(%d)
  ["refresh_rate"]=>
  int(%d)
}
SDL_GetCurrentDisplayMode(0):object(SDL_DisplayMode)#2 (4) {
  ["format"]=>
  int(%d)
  ["w"]=>
  int(%d)
  ["h"]=>
  int(%d)
  ["refresh_rate"]=>
  int(%d)
}
string(%d) "SDL_DisplayMode(%s,%d,%d,%d)"
NULL
int(0)
NULL
int(-1)
NULL
Done
---- ACTUAL OUTPUT
== VideoDriver ==
int(4)
string(3) "x11"
bool(false)
== Video OFF ==
bool(false)
int(0)
== Video ON ==
SDL_VideoInit():int(-1)
SDL_GetCurrentVideoDriver():bool(false)
SDL_GetNumVideoDisplays():int(0)
SDL_GetDisplayName(0):bool(false)
SDL_GetDisplayBounds(0):int(-1)
NULL
SDL_GetNumDisplayModes(0):int(-1)
SDL_GetDisplayMode(0,0):bool(false)
SDL_GetDesktopDisplayMode(0):bool(false)
SDL_GetCurrentDisplayMode(0):bool(false)
string(0) ""
NULL
Segmentation fault (core dumped)

Termsig=11
---- FAILED
