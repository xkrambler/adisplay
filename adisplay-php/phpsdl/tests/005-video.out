== VideoDriver ==
int(4)
string(3) "x11"
bool(false)
== Video OFF ==
bool(false)
int(0)
== Video ON ==
SDL_VideoInit():int(-1)
SDL_GetCurrentVideoDriver():bool(false)
SDL_GetNumVideoDisplays():int(0)
SDL_GetDisplayName(0):bool(false)
SDL_GetDisplayBounds(0):int(-1)
NULL
SDL_GetNumDisplayModes(0):int(-1)
SDL_GetDisplayMode(0,0):bool(false)
SDL_GetDesktopDisplayMode(0):bool(false)
SDL_GetCurrentDisplayMode(0):bool(false)
string(0) ""
NULL
Segmentation fault (core dumped)

Termsig=11